### PR TITLE
Rewrote the "Getting Started" section

### PR DESCRIPTION
**Pull request by webmozart**
| Q | A
| --- | ---
| Fixed tickets | -
| License | MIT

#### What's in this PR?

This PR rewrites the "Getting Started" section to be shorter and easier to read and follow. It also changes "Getting Started" to be based on the Sulu Minimal edition.

#### Why?

The current "Getting Started" section explains a lot of information that isn't really needed for getting a project up and running with Sulu. Since this is the first point of interaction with Sulu for a new user, it should provide the best possible experience.

### CODE REVIEW

File: book/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+The content management part of Sulu is built upon *webspaces*. Each of these
+webspaces configure a content tree. Each content tree may contain translations
+for different locales.
+
+The default webspace configuration is located in
+``app/Resources/webspaces/example.com.xml``. Rename this file so that it matches
+the name of your project.
+
+To get started, you only need to modify the ``<key>`` in that file. Change it
```
Comment (danrot) [inline book/getting-started.rst]: Maybe we should also mention `<name>` here, and explain that it is the text being displayed in the UI?

File: book/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+to the name of your project:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.0.xsd">
```
Comment (danrot) [inline book/getting-started.rst]: We should probably go for `webspace-1.1.xsd`, since it is the latest version.

File: book/getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    recommend to decide what key to use before you build the database in the
+    next step.
+
+We'll :doc:`return to webspaces <webspaces>` later in this book.
+
+Setup the Database
+------------------
+
+Next we'll setup a database for Sulu. You can use Sulu with any of the
+database backends `supported by Doctrine DBAL`_.
```
Comment (danrot) [inline book/getting-started.rst]: Isn't that quite a big promise? Actually we already know that it is not working with sqlite.

File: book/getting-started.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+    The :doc:`parameter reference <../reference/parameters>` contains more
+    information about each of the parameters in this file.
+
+When you're done with the configuration, populate the database with Sulu's
+default data:
+
+.. code-block:: bash
+
+    bin/adminconsole sulu:build dev
```
Comment (danrot) [inline book/getting-started.rst]: Shouldn't we explain here that a standard user is created with that command? Wouldn't be ideal if every Sulu installation has a `admin/admin` user 😃

File: book/getting-started.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+.. code-block:: bash
+
+    app/adminconsole server:start
+    app/websiteconsole server:start
+
+You can access the administration interface via http://127.0.0.1:8000/admin.
+The default user and password is "admin".
+
+The web frontend can be found under http://127.0.0.1:8001.
```
Comment (danrot) [inline book/getting-started.rst]: We should get https://github.com/sulu/sulu/issues/2423 really fixed if we want to advocate this in the documentation...

File: book/getting-started.rst [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+and the website. Each application is optimized for its purpose. The applications
+can be managed with the command line tools ``app/adminconsole`` (for the
+administration) and ``app/website`` (for the website).
+
+We'll run one server for each application:
+
+.. code-block:: bash
+
+    app/adminconsole server:start
+    app/websiteconsole server:start
```
Comment (danrot) [inline book/getting-started.rst]: These files are located in the `bin` folder

File: cookbook/web-server/nginx.rst [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      # pass the PHP scripts to FastCGI server from upstream phpfcgi
+      location ~ ^/(website|admin|app|app_dev|config)\.php(/|$) {
+          include fastcgi_params;
+          fastcgi_pass unix:/var/run/php5-fpm.sock;
+          fastcgi_buffers 16 16k;
+          fastcgi_buffer_size 32k;
+          fastcgi_split_path_info ^(.+\.php)(/.*)$;
+          fastcgi_param  SCRIPT_FILENAME $document_root$fastcgi_script_name;
+          fastcgi_param  SYMFONY_ENV dev;
+          fastcgi_param  HTTPS off;
```
Comment (danrot) [inline cookbook/web-server/nginx.rst]: @chirimoya What should we do about this now? We discussed about removing this line.

**Comment by danrot** [comment]
@webmozart Looks like there is still a wrong link (at least the tests say that).

File: book/getting-started.rst (hunk 3 above)
Comment (webmozart) [inline book/getting-started.rst]: Ok, any other exceptions?

File: book/getting-started.rst (hunk 1 above)
Comment (webmozart) [inline book/getting-started.rst]: Good point, fixed!

File: book/getting-started.rst (hunk 2 above)
Comment (webmozart) [inline book/getting-started.rst]: Good catch

File: book/getting-started.rst (hunk 3 above)
Comment (danrot) [inline book/getting-started.rst]: I know that Sulu works with MySQL and PostgreSQL, can't guarantee for anything else, except that it doesn't run with sqlite.

Comment (webmozart) [inline book/getting-started.rst]: Added a table now.

File: book/getting-started.rst (hunk 4 above)
Comment (webmozart) [inline book/getting-started.rst]: Yes, good idea! :)

File: book/getting-started.rst (hunk 5 above)
Comment (webmozart) [inline book/getting-started.rst]: Yes! :)

File: book/getting-started.rst (hunk 6 above)
Comment (webmozart) [inline book/getting-started.rst]: :innocent:

**Comment by webmozart** [comment]
@danrot Thanks for the feedback :) The PR is updated now.

**Comment by danrot** [comment]
Great :) Lets wait for the ACK of @chirimoya before merging.

**Comment by chirimoya** [comment]
ACK

File: cookbook/web-server/nginx.rst (hunk 7 above)
Comment (chirimoya) [inline cookbook/web-server/nginx.rst]: remove `fastcgi_param  HTTPS off;`

**Pull request merged**